kano_updater/commands/install.py

```diff
@@ -313,34 +313,34 @@ def install_standard(progress, status):
         status.state = UpdaterStatus.UPDATES_DOWNLOADED
         status.save()
 
-        logger.info(_('The updater has been updated, relaunching.'))
+        logger.info(_("The updater has been updated, relaunching."))
         progress.relaunch()
         return False
 
     progress.start('preupdate')
     try:
         preup.run()
     except Exception as err:
-        logger.error('The pre-update scenarios failed.')
-        logger.error(str(err))
-        progress.abort(_('The pre-update tasks failed.'))
+        logger.error("The pre-update scenarios failed.")
+        logger.error(err.encode('utf-8'))
+        progress.abort(_("The pre-update tasks failed."))
```

Comment:
It looks like the `abort` methods here are going to do a call to the logger with translated strings.